src/init.lua

```diff
@@ -20,6 +20,11 @@ local IS_SERVER = RunService:IsServer()
 
 export type Settings = typeof(Settings)
 
+local QuickSetupUtility = require(Utility.QuickSetup)
+local Dispatcher = require(script.Dispatcher)
+local Simulation = require(script.Simulation)
+local SnapshotsUtility = require(Utility.Snapshots)
```

Comment:
I only meant for the `QuickSetupUtility` to be moved, moving the rest will cause them to error as they expect the characters and or visuals folder to exist which someone using the quick setup option might not have statically in their workspace.